doc/release-notes/5.1.1-release-notes.md

```diff
@@ -0,0 +1,59 @@
+# Dataverse 5.1.1
+
+This minor release adds important optimizations to installations running on AWS S3. It is recommended that 5.1.1 be used in production instead of 5.1.
```

Comment:
'scaling improvements' instead of 'optimizations'?

Reply:
updated in https://github.com/IQSS/dataverse/pull/7319/commits/49af1973788b47aa53b68d0fe7c5741da4202c6c